src/utils/ContractUtils.ts

```diff
@@ -82,3 +91,48 @@ export async function calculateEstimatedGas(
     .estimateGas({ from }, (err, estGas) => (err ? GASLIMIT_DEFAULT : estGas))
   return estimatedGas
 }
+
+/**
+ * Send the transation on chain
+ * @param {string} from account that calls the function
+ * @param {any} estGas estimated gas for the transaction
+ * @param {Web3} web3 web3 objcet
+ * @param {Function} functionToEstimateGas function that we need to send
+ * @param {...any[]} args arguments of the function
+ * @return {Promise<any>} transaction receipt
+ */
+export async function sendTx(
+  from: string,
+  estGas: any,
+  web3: Web3,
+  config: Config,
```

Comment:
is there anything else used from this object than `config.gasFeeMultiplier`? If not, then this should be the param, and not the whole config object

Reply:
yep thought about having that instead of the whole object, will update this and the calls 😃 , think i was not sure if we need something else from config at the development time